### ISSUE TITLE
strip-block produces different output in Smarty v3.1.32 

### ISSUE THREAD

**Issue by rvock** [smarty-php/smarty#436]
Smarty v3.1.32 had an undocumented breaking change for the `{strip}`-Block:

```
{strip}
	<li>Code  <- two spaces</li>

	{*<!-- Comment -->*}
	<li>Code</li>
{/strip}
```

This Template would lead to the following HTML in Smarty v3.1.31:
```
<li>Code  <- two spaces</li><li>Code</li>
```

All linebreaks and multiple whitespace at start or end would be removed.

With Smarty v3.1.32 there is additional whitespace:
```
<li>Code  <- two spaces</li>	<li>Code</li>
```

This can break a lot of templates for menus, if they rely on the missing whitespace between the li-tags.

Just a note: If the comment is removed or there is an additional linebreak after the comment, everything works fine.

**Comment by uwetews** [comment]
The fix is now in the master branch and will later be included in 3.1.33

**Comment by biesbjerg** [comment]
Was this included in 3.1.33?

```smarty
{strip}

	{* Evaluate and output header template *}
	<?xml version="1.0"?><rss version ="2.0" xmlns:g="http://base.google.com/ns/1.0">
	<item>
		<description>...</description>
	</item>

{/strip}
```

Produces:

3.1.31:
```xml
<?xml version="1.0"?><rss version ="2.0" xmlns:g="http://base.google.com/ns/1.0"><item><description>...</description></item>

```

3.1.33:
```xml
	<?xml version="1.0"?><rss version ="2.0" xmlns:g="http://base.google.com/ns/1.0"><item><description>...</description></item>

```
Notice the leading tab. If I remove the comment from the template there is no leading tab.
